daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -15,6 +15,156 @@
     set_left_brightness as _set_left_brightness, set_right_brightness as _set_right_brightness
 
 
+class RazerLanceheadWireless(__RazerDeviceSpecialBrightnessSuspend):
```

Comment:
As the two devices are actually one and should support exactly the same things, you can e.g. make the Wireless version inherit from the Wired one and just overwrite the PID and potential other relevant parts.

Reply:
Should be good now.